### PR TITLE
docs: replace sapper docs with shiki docs

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

File: packages/site/src/routes/_docs.svtext [hunk 1]
```diff
@@ -662,21 +662,34 @@ Frontmatter also interacts with layouts, you can find more details in the [Layou
...
-- [Webpack](https://github.com/shiryel/sapper-mdsvex-template-webpack)
+const theme = 'github-dark';
+const langs = ['javascript', 'typescript'];
+const highlighter = await createHighlighter({
+	themes: [theme],
+	langs
+});
+for (const lang of langs) {
+	await highlighter.loadLanguage(lang);
+}
```
Comment (ghostdevv) [inline packages/site/src/routes/_docs.svtext:681]:
```suggestion
const langs = ['javascript', 'typescript'];
const theme = 'github-dark';

const highlighter = await createHighlighter({
	themes: [theme],
	langs
});

for (const lang of langs) {
	await highlighter.loadLanguage(lang);
}
```

might make it a tiny bit more readable

Comment (honnip) [inline packages/site/src/routes/_docs.svtext:681]: I don't think it's not need to load the languages again because it was created with that

Comment (pngwn) [reply packages/site/src/routes/_docs.svtext:681]: @benmccann do we want to load the languages on demand in this example or do you think it is better to keep it simple?

Comment (benmccann) [inline packages/site/src/routes/_docs.svtext:681]:
```suggestion
```

File: packages/site/src/routes/_docs.svtext [hunk 2]
```diff
@@ -662,21 +662,31 @@ Frontmatter also interacts with layouts, you can find more details in the [Layou
...
-  ```bash
-  npx degit "pngwn/sapper-mdsvex-template" my-app
-  ```
-- [Webpack](https://github.com/shiryel/sapper-mdsvex-template-webpack)
+const theme = 'github-dark';
+const langs = ['javascript', 'typescript'];
+const highlighter = await createHighlighter({
+	themes: [theme],
+	langs
+});
```
Comment (benmccann) [inline packages/site/src/routes/_docs.svtext:678]:
```suggestion
const highlighter = await createHighlighter({
	themes: [theme],
	langs: ['javascript', 'typescript']
});
```

File: packages/site/src/routes/_docs.svtext (hunk 1 above)
Comment (benmccann) [reply packages/site/src/routes/_docs.svtext]: great catch @honnip! that was in the example I copied from, but I took a look at the docs and you look to be correct that it's unnecessary, so I've removed it

Comment (benmccann) [reply packages/site/src/routes/_docs.svtext]: obsoleted by other changes

**Comment by benmccann** [comment]
Thanks for all the fixes!

**Comment by pngwn** [comment]
https://mdsvex.pngwn.io/docs#integrations !!

**Pull request merged**